MyGet.ps1

```diff
@@ -0,0 +1,43 @@
+# Performs the build for MyGet and uploads the generated NuGet packages to
+# the "google-dotnet" feed.
+#
+# MyGet's build syncs the repo and subrepos, to build manually be sure to:
+# git submodule update --init ClientGenerator
+
+function PrintHeader($message) {
+    Write-Host "****************************************"
+    Write-Host $message
+    Write-Host "****************************************"
+}
+
+Write-Host ".NET Client Libraries, build shard $Env:BuildShard"
+
+# TODO(chrsmith): Document why we require VS v14 to build.
```

Comment:
VisualStudioVersion needs to be defined and set to a version of VS installed on the system. I think 12 would work as well as 14. Heck, maybe even 10?

http://blogs.msdn.com/b/webdev/archive/2012/08/22/visual-studio-project-compatability-and-visualstudioversion.aspx